config.yaml

```diff
@@ -1,8 +1,9 @@
-#------------------------------------------------------------
-# Values for this lesson.
-#------------------------------------------------------------
+---
```

Comment:
[YAML lint](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_start): the [YAML standard](https://yaml.org) requires that each YAML document begins with the `---` (comments or magics above are OK).

Reply:
from what I understand, it's not a requirement per-se, but it could optionally be added as a requirement for a given implementation and is left to opinion. 

Neither GitHub nor any of the handlers we use enable the `document-start` requirement, so it would be better to configure the linter to use `document-start: disable` (as we do in [glosario](https://github.com/yaml/yaml-spec/issues/73))

Reply:
While it is not regularly enforced, but it is still part of the YAML standard. Since this repository is a template for others to use, I am in favor of stricter adherence to standards; downstream repositories can relax if they so choose.

Also, and perhaps more convincingly, the "---" makes a tidy delineation between the top-level comment explaining what a file is for, and the implementation and section-specific comments in the data itself. For example, `config.yaml` currently dives right in to "Values for this lesson" which is unclear. We could (should) lead in, along the lines of

```yaml
# Configuration and template parameters for your lesson.
# Values modified here will propagate through the website
# when you build it, setting the appropriate
# logo/theme, contact info, and episode structure.
---

# -----------------------
# Values for this lesson.
# -----------------------
...
```

Reply:
Can you open an issue in {sandpaper} for this? I will revisit after the transition in May.